exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -532,30 +536,45 @@ func (b *BTCMarkets) GetFeeByType(feeBuilder *exchange.FeeBuilder) (float64, err
 
 // GetActiveOrders retrieves any orders that are active/open
 func (b *BTCMarkets) GetActiveOrders(req *order.GetOrdersRequest) ([]order.Detail, error) {
-	var resp []order.Detail
-	var tempResp order.Detail
-	var tempData []OrderData
 	if len(req.Currencies) == 0 {
-		allPairs := b.GetEnabledPairs(asset.Spot)
+		allPairs := b.GetAvailablePairs(asset.Spot)
```

Comment:
We should keep this as GetEnabledPairs, so that we're only interested in the ones the user has enabled (otherwise they can set it in req.Currencies). Our order manager will use enabled pairs in a future update.